pkg/pb/overlay.proto

```diff
@@ -13,6 +13,7 @@ service Nodes {
     rpc Query(QueryRequest) returns (QueryResponse);
     rpc Ping(PingRequest) returns (PingResponse);
     rpc RequestInfo(InfoRequest) returns (InfoResponse);
+    rpc RequestGraph(GraphRequest) returns (GraphResponse);
```

Comment:
Adding a public rpc call for this concerns me. I would like to avoid adding things here that are not essential parts of the API. Only having it internal only should suffice.

Reply:
oh, yeah, the original ticket said that we should be able to ask any node, but it looks like i changed the ticket on april 3 to be specifically only the inspector and only the internal port. so yeah, i think this should be the set of inspector-only methods? i don't know where those are defined though

Reply:
The PR already added it to the inspector.

Reply:
oh, great!

Reply:
I'm not sure I understand well what you are saying. Does it mean we shouldn't be able to call "inspector kad routing-graph <node-id>?

Reply:
@Fadila82 As far as I get it, the inspector should not be able to ask for the routing graph of any other node except the current node, i.e. the one we are connecting the inspector to.

So we should be able to call only `inspector kad routing-graph` (without `node-id` argument), which will return the routing graph of the current node.

Thus, you should not need the `Nodes.RequestGraph` rpc method. The `KadInspector.DrawTable` should be enough to achieve this.